### DISCUSSION TITLE
docs/core/gview-funcs-index

### DISCUSSION THREAD

**Discussion by giscus** [gogf/gf-site#101] [General]
# docs/core/gview-funcs-index

Golang标准库中的基础函数及其在GoFrame框架中的改进使用。通过实用示例展示了如何在不同模板函数中进行参数传递，以及如何灵活调用函数。详细解读了and、or、not、call、index等函数的用法，以及GoFrame框架中对eq/ne/lt/le/gt/ge比较函数的自动类型转换优化。

https://goframe.org/docs/core/gview-funcs-index

**Comment by xiaoyi510** [comment]
能否兼容 基础的 +-x÷

**Reply by gqcn** [reply]
在这里 https://goframe.org/docs/core/gview-funcs-builtin 有对应的 `plus/minus/times/divide`模板内置函数。
